packages/apps/explorer/src/utils/condenseStrings.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Shortens long sections in a string by keeping the first 5 and last 5 characters.
```

Comment:
i think we have a `maskValue` function in the kode-ui
is this not the same?

Reply:
It's slightly different. 
It's unused atm but I wanted to keep it as it's the start for a component that could use the `hidden=until-found`.
Also, the MaskValue is for a single value, this is for a whole string.

E.g.: 

```
{ "code": "(n_f17eb6408bb84795b1c871efa678758882a8744a.coin-faucet.create-and-request-coin "k:554754f48b16df24b552f6832dda090642ed9658559fef9f3ee1bb4637ea7c94" (read-keyset "new_keyset") 20.0)" }

{ "code": "(n_f17…8744a.coin-faucet.create-and-request-coin "k:55475…a7c94" (read-keyset "new_keyset") 20.0)" }

```